crates/uv-python/src/version_files.rs

```diff
@@ -70,6 +70,7 @@ impl PythonVersionFile {
                         let trimmed = line.trim();
                         !(trimmed.is_empty() || trimmed.starts_with('#'))
                     })
+                    .flat_map(|line| line.split_whitespace())
```

Comment:
```suggestion
```

I have tested again, and it looks like the current release of pyenv put each entry in a separate line. 

Even if it works with space separated list in file, it should be enough to call `pyenv local list_of_pytho_vrsion` again to get it working